echobash/codebreaker
# Create livewire components

This PR ensures that whenever `increment` or `decrement` button is called, a click music is played.

## Testing instructions -
- Go to Counter page
- Click increment button - A click sound should be heard.
- Click decrement button - A click sound should be heard.
- Click counter number button - No any sound should be heard.
